pkg/arvo/app/chat-hook.hoon

```diff
@@ -85,8 +87,20 @@
     =/  old  !<(versioned-state old-vase)
     =|  cards=(list card)
     |-
-    ?:  ?=(%8 -.old)
+    ?:  ?=(%9 -.old)
       [cards this(state old)]
+    ?:  ?=(%8 -.old)
```

Comment:
The people encountering this issue haven't successfully processed the last state migration, so wouldn't their state still be at `%7`?

Reply:
Yes, hence the change to the logic in the `%7` upgrade case so as not to crash when encountering a chat without a corresponding group.

Reply:
You could probably get away with including this in the upgrade logic for `%7` then, considering that upgrade hasn't happened yet for everyone who has syncs that would be deleted by this.

Also you're deleting from `synced` but not `allow-history`? I get the latter is only for things we're hosting, but the former contains those too, right?